server/auth.go

```diff
@@ -368,6 +369,93 @@ func (c *client) matchesPinnedCert(tlsPinnedCerts PinnedCertSet) bool {
 	return true
 }
 
+// JWT Tag value for Account JWT Tag, enabling on-boarding services
+const featureTagOnboard = "nats-onboard-jwt"
+const onboardSubjFmt = "$jwt.request.%s"
+
+func (s *Server) onboardingCallout(c *client, opts *Options) bool {
+	var acc *Account
+	if c.opts.Account == _EMPTY_ {
+		// check default (system account) callout
+		acc = s.SystemAccount()
+		if acc == nil {
+			c.Debugf("Authentication requires a user JWT - Token Request failed - Account Missing")
+			return false
+		}
+	} else {
+		// check account listed
+		var err error
+		acc, err = s.fetchAccount(c.opts.Account)
+		if err != nil {
+			c.Debugf("Authentication requires a user JWT - Token Request Account %q lookup failed: %s", c.opts.Account, err)
+			return false
+		}
+	}
+	acc.mu.RLock()
+	signupName := fmt.Sprintf("Account: %s/%s", acc.Name, acc.nameTag)
+	featOnboard := acc.tags.Contains(featureTagOnboard)
+	acc.mu.RUnlock()
+	if !featOnboard {
+		c.Debugf("Authentication requires a user JWT")
```

Comment:
Shouldn't server debug report that Account blah-blah not enabled for external onboarding service?  Otherwise, not helpful to discriminate and fix.